src/custom-elements/src/components/carousel/carousel.tsx

```diff
@@ -207,8 +211,13 @@ export class Carousel {
     remainingItems: number,
     itemWidth: number
   ) {
+    const gap = this._getGap();
+
     this.remainingSpace = Math.abs(
-      this.fullScrollLength - (this.maxItemsShown + remainingItems) * itemWidth
+      this.fullScrollLength -
+        (this.maxItemsShown + remainingItems) * itemWidth -
```

Comment:
this.maxItemsShown + remainingItems --> this is used twice here so these can be merged into a new const